.circleci/tests/custom_values_charts_test.go

```diff
@@ -0,0 +1,37 @@
+package tests
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+func (s *helmRepoSuite) TestChartWithCustomValues() {
+	testFilesDirectory := path.Join("testfiles", "custom_values")
```

Comment:
Let's change the folder structure to be:
testfiles/<chart_name>/testName.yaml
testfiles/<chart_name>/testName2.yaml

Also let's split golden files to folder per chart